contracts/PvpArena.sol

```diff
@@ -836,4 +845,82 @@ contract PvpArena is Initializable, AccessControlUpgradeable {
         //this is not final, but processing the loser will update the ranks leaving this player in the 4th position, which will be quickly replaced by other players
         processLoser(characterID);
     }
+
+    /// @dev distributes the ranking rewards pool to top players
+    function distributeSeasonRewards() external restricted {
```

Comment:
This name could be a bit missleading since you are assigning rather than distribute?